sdk/eventhub/event-hubs/package.json

```diff
@@ -62,7 +62,7 @@
     "unit-test": "npm run unit-test:node && npm run unit-test:browser"
   },
   "dependencies": {
-    "@azure/amqp-common": "^1.0.0-preview.5",
+    "@azure/amqp-common": "^1.0.0-preview.9",
```

Comment:
Oof, that's a big leap forward in version!

Reply:
Didn't we learn something around not using ^ when dealing with preview versions?

Reply:
We aren’t releasing service bus or event hubs from this branch. I only updated the dep to appease rush.